tox.ini

```diff
@@ -29,17 +28,14 @@ commands =
 [testenv:quality]
 basepython=python3.6
 deps=
-  pytest==3.0.2
-  flaky
+    -rrequirements-test.txt
 commands=
     python -m pytest tests/quality/
 
 [testenv:oldpy27]
 basepython=python2.7.3
 deps=
-  pytest==3.0.2
-  flaky
-  uncompyle6
```

Comment:
We’ve lost the uncompyle6 dependency – because it’s no longer used anywhere, yes?